internal/backends/gl/glwindow.rs

```diff
@@ -697,7 +696,7 @@ impl PlatformWindow for GLWindow {
             },
         );
 
-        result / scale_factor
+        Rect::new(result / scale_factor, Size::new(1.0, font.height() / scale_factor))
```

Comment:
i initially did that, but then then the cursor width is scaled down as well and then i had to make the width 2.0 to counteract and while this may save some cpu cycles it seemed not very intuitive.